### PR TITLE
Fixes for 0.2 release

### PR DESCRIPTION
**Pull request by tbekolay**
This does a few minor things in preparation for the 0.2 release. Specifically:

- Convert `.md` files to` .rst` for consistency with Nengo and README.rst.
- Update license with minor nits found by Pete (as done in Nengo)
- Update version numbering scheme to conform to PEP440
- Edit some parts of the README
- Make a `nengo` script and deprecate the `nengo_gui` script

See individual commits for more information.

**Before this PR is merged** the following contributors need to sign ``CONTRIBUTORS.rst``, as we missed them when merging pull requests.

- [x] @pblouw 
- [x] @rsimmons1 

To sign the file, please make a commit adding yourself to the ``CONTRIBUTORS.rst`` file. That commit can be pushed to this branch (or if you prefer, you make a separate branch and PR for it).

Please read that file, as well as the assignment agreement in ``LICENSE.rst``. If you're unsure about what the agreement means, please ask either me or @tcstewar!

### CODE REVIEW

**Comment by tbekolay** [comment]
Thanks to those who signed the contributors doc! @tcstewar, can you review this now?

File: nengo_gui/version.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
 """Nengo GUI version information.
 
 We use semantic versioning (see http://semver.org/).
-Additionally, '-dev' will be added to the version unless the code base
-represents a release version. Commits for which the version doesn't have
-'-dev' should be git tagged with the version.
+and confrom to PEP440 (see https://www.python.org/dev/peps/pep-0440/).
```
Comment (tcstewar) [inline nengo_gui/version.py]: "confrom" should be "conform"

**Comment by tcstewar** [comment]
Other than the one typo, looks great to me!

**Comment by tbekolay** [comment]
Typo fixed!

**Comment by tcstewar** [comment]
> Typo fixed!

LGTM!  All set for release -- thank you everyone!

**Comment by tbekolay** [comment]
Cool! Can you merge?

**Comment by tcstewar** [comment]
Merged!

And I think I'd like this to be the last merge into master done the lazy way, and in the future I should do merges the correct way that they're being done in ```nengo```.....  :)

**Pull request merged**